docs/advanced-topics/checksum-support.rst

```diff
@@ -94,15 +94,15 @@ or `Azure <https://docs.microsoft.com/en-us/azure/container-registry/container-r
 Tools
 -----
 As noted in the table above, Docker image checksums are grabbed on refresh and should work as long as the image is from Quay.io, Docker Hub,
-or GitLab. It's also important to note that this is done for the Docker image registered on the tool through Dockstore and not necessarily
-the one included within the descriptor file itself.
+Amazon ECR, or GitLab. It's also important to note that this is done for the Docker image registered on the tool through Dockstore and not necessarily
+the one referenced within the descriptor file itself.
 
-Workflows
----------
-For workflows, Docker image checksums are grabbed on snapshot. However, the Docker images we can retrieve from descriptor files
+Workflows and GitHub App Tools
+------------------------------
+For workflows and GitHub App tools, Docker image checksums are grabbed on snapshot. However, the Docker images we can retrieve from descriptor files
```

Comment:
Not required, but this could link out to a GitHub App Tools documentation page (as it's not standard terminology).

Reply:
I don't think we have a dedicated GitHub App Tools documentation page right now